tests/test_cases/test_skipped/test_skipped.py

```diff
@@ -0,0 +1,18 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pathlib
+
+import cocotb
+from cocotb import triggers
+
+skipped_file_name = "ran_skipped_test~"
+
+
+@cocotb.test(skip=True)
+def test_skipped(dut):
+    """ Touch a file so we can check that this test has ran."""
+    yield triggers.NullTrigger()
+    dut._log.info("Ran skipped test manually.")
+    pathlib.Path(skipped_file_name).touch()
```

Comment:
```suggestion
@cocotb.test(skip=True)
async def test_skipped(dut):
    """ Touch a file so we can check that this test has ran."""
    pathlib.Path(skipped_file_name).touch()
```

Reply:
Aha, thanks. I'm still not fully used to the new ```async```-based way of doing things.

Updated and rebased with both changes.